src/test/java/org/apache/drillui/test/framework/steps/restapi/RestSecuritySteps.java

```diff
@@ -28,28 +29,12 @@ private RestSecuritySteps() {
   }
 
   public static SessionFilter getDefaultSession() {
-    return TestProperties.getBool("SECURE_DRILL") ?
-        login(
-            TestProperties.get("ADMIN_USER1_NAME"),
-            TestProperties.get("ADMIN_USER1_PASSWORD")) :
-        new SessionFilter();
+    return PropertiesConst.SECURE_DRILL ? login(PropertiesConst.ADMIN_1_NAME, PropertiesConst.ADMIN_1_PASSWORD) : new SessionFilter();
```

Comment:
Please roll back the previous formatting.

Reply:
Done